src/openrct2/config/Config.cpp

```diff
@@ -706,6 +706,17 @@ namespace Config
             }
         }
 
+        //        static constexpr u8string_view rctcSearchLocations[] = {
```

Comment:
This bit still has to be filled in with the appropriate directories where RCTC can reside on Windows and macOS. For this, I’ll need some help from Windows users.

Reply:
In the case of Steam on Windows this will be `Steam\steamapps\common\RollerCoaster Tycoon Classic`